samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -72,6 +72,39 @@
           // adjust the height of the iframe to accomodate the expansion from the accordion
           $("body").height($("body").outerHeight() + 900);
 
+          checkNav = function() {
+              QuesFormatRadios = ["assessmentSettingsAction\\:assessmentFormat\\:0", "assessmentSettingsAction\\:assessmentFormat\\:1", "assessmentSettingsAction\\:assessmentFormat\\:2"];
+
+              if ($("#assessmentSettingsAction\\:itemNavigation\\:0:checked").val() !== undefined) {
+                  enabled = false;
+              } else {
```

Comment:
better to just set enabled = true on line 77 and then you have an initialized variable

Reply:
Not a problem; I'll fix this up later this evening.